### ISSUE TITLE
Kitematic cannot see VirtualBox

### ISSUE THREAD

**Issue by baynezy** [docker/kitematic#2662]
### Expected behavior

Kitematic should load the GUI and I should be able to manage Docker containers

### Actual behavior

I see the same screen as described in #2499. However, when I use those steps outlined they do not resolve it.

You can see [an example](https://www.screencast.com/t/542zF0cCKvU). I assume the `Invalid API Key` error is for BugSnag and not Kitematic

### Information about the Issue

- Windows 7 Enterprise
- VirtualBox 5.1.22
- Docker Toolbox
- Kitematic (latest version)

### Steps to reproduce the behavior

I am not certain this will help you reproduce but Kitematic was working fine until I upgraded from VirtualBox 4.3.40 to 5.1.22.

**Comment by FrenchBen** [comment]
I appreciate the screencast, but adobe flash player isn't something I'm going to install to view it. 
Although not ideal, can you do a clean install?
Additionally what does the `docker-machine create -D default` output?

**Comment by DylenTurnbull** [comment]
I ran into a similar issue and the quick and dirty fix is quit docker and then delete the Kitematic folder at C:\Users\YOUR USER FOLDER\AppData\Roaming, restart docker and run Kitematic and log in again.  I saw the issue after installing docker then copying Kitematic in place it ran fine then I foolishly installed the toolbox with  virtual box and Kitematic no longer worked.  I suspect when changes are made the are not getting set properly in the apps settings in AppData\Roaming.
